montrek-software/montrek
# 152 write view that returns a list with these

## How to test:

- Go to `http://127.0.0.1:8000/montrek_example/b/list` and create a few Example Bs:

![image](https://github.com/chrishombach/montrek/assets/29869670/240f8934-9928-4864-b51c-a87f094a6831)

- Go to `http://127.0.0.1:8000/montrek_example/a/list` and create a Example A:

 
![image](https://github.com/chrishombach/montrek/assets/29869670/156dd3ef-854b-4c2c-b6ed-92185e1cd068)

- See this item in history:

![image](https://github.com/chrishombach/montrek/assets/29869670/833f9778-6085-453e-a3c2-6f98e0c4028c)

- Update this item:

![image](https://github.com/chrishombach/montrek/assets/29869670/d7702887-11c9-4cf6-a033-ba45ae6dd69d)

- See history

![image](https://github.com/chrishombach/montrek/assets/29869670/a27ce788-afa6-4b69-b643-15e287739aa5)
